.github/workflows/verify.yaml

```diff
@@ -18,3 +18,16 @@ jobs:
           java-version: '17'
           distribution: 'temurin'
       - run: mvn clean verify
+  verify_frontend:
+    name: Verify Frontend
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Node set-up"
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+      - name: Install dependencies
+        run: cd frontend && npm install
+      - name: Build
+        run: cd frontend && npm run build -- --mode=production
```

Comment:
You can check out the [working-directory](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory) keyword

Reply:
Good catch, done